nordic/trezor/boards/arm/t3w1_revA_nrf52832/t3w1_revA_nrf52832.dts

```diff
@@ -22,6 +22,7 @@
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
 		zephyr,code-partition = &slot0_partition;
+                zephyr,bt-hci = &bt_hci_controller;
```

Comment:
please add these changes to the t3w1_d1_nrf538322 board too

Reply:
Added to the the old devboard as well.